lib/Mojo/Reactor/Epoll.pm

```diff
@@ -22,6 +21,14 @@ sub io {
 	return $self->watch($handle, 1, 1);
 }
 
+sub again {
+	my ($self, $id, $after) = @_;
+	croak 'Timer not active' unless my $timer = $self->{timers}{$id};
+	$after //= $timer->{after};
```

Comment:
I think that this should actually assign to `$timer->{after}` if `$after` is defined to match the implementation in Mojo::Reactor::Poll. I guess this would only matter if `again` is called again on the same timer without a new `$after`.

Reply:
Yeah that makes sense